.travis.yml

```diff
@@ -5,43 +5,27 @@ language: bash
 services:
   - docker
 
-env:
-  global:
-    - ARTIFACTORY_PRIVATE_USERNAME=private-reader
-    - SONARQUBE_BUILD_NUMBER=8.0
-
 script:
   - 'if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
         ./run-public-image-tests.sh;
     fi'
   - docker build "7/community" --tag sonarqube:7
   - ./run-tests.sh "sonarqube:7"
   - >-
-    SQ_VERSION="$SONARQUBE_BUILD_NUMBER"
-    ZIP_SERVER="repox.jfrog.io"
-    ZIP_URL="https://repox.jfrog.io/repox/sonarsource/org/sonarsource/sonarqube/sonar-application/$SONARQUBE_BUILD_NUMBER/sonar-application-$SONARQUBE_BUILD_NUMBER.zip"
-    ZIP_USERNAME="$ARTIFACTORY_PRIVATE_USERNAME"
-    ZIP_PASSWORD="$ARTIFACTORY_PRIVATE_PASSWORD"
-    ./8/build-docker-from-private-repo.sh community
-    --tag sonarqube:8
+    docker build "8/community"
+    --build-arg SONARQUBE_VERSION="8.2.0.32423"
+    --build-arg SONARQUBE_ZIP_URL="https://repox.jfrog.io/repox/sonarsource-public-dev/org/sonarsource/sonarqube/sonar-application/8.2.0.32423/sonar-application-8.2.0.32423.zip"
+    --tag "sonarqube:8"
```

Comment:
This is a test, it doesn't matter. We could even skip adding the tag and reference by the generated one. Besides, "8" is a valid tag for this image and all of the subsequent 8.x images because each time we will be marking the next "8.x" as "8" too.